packages/launch/src/focused-launch/plan-details/index.tsx

```diff
@@ -3,22 +3,80 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import { Link } from 'react-router-dom';
+import * as React from 'react';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { Plans } from '@automattic/data-stores';
+import PlansGrid from '@automattic/plans-grid';
+import { Title, SubTitle } from '@automattic/onboarding';
+import { useHistory } from 'react-router-dom';
 
 /**
  * Internal dependencies
  */
-import { Route } from '../route';
+import { LAUNCH_STORE } from '../../stores';
+import GoBackButton from '../go-back-button';
 
 import './style.scss';
 
 const PlanDetails: React.FunctionComponent = () => {
+	const domain = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedDomain() );
+	const selectedPlan = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedPlan() );
+	const history = useHistory();
+
+	const { updatePlan } = useDispatch( LAUNCH_STORE );
+
+	const hasPaidDomain = domain && ! domain.is_free;
+
+	const handleSelect = ( planSlug: Plans.PlanSlug ) => {
```

Comment:
@ciampo used a [different approach](https://github.com/Automattic/wp-calypso/pull/47404/files#diff-e51e01d27d081dc244f31143d0841e80254d49c01cee0cb7135bb8df8fa22a62R43-R45) to "Going back" we should unify these at some point. What do you think?